script-test/subtitles/imscsubtitles.js

```diff
@@ -283,7 +283,7 @@ require(
           loadUrlStubResponseText = '<?xml version="1.0" encoding="utf-8" extra property="something"?><tt xmlns="http://www.w3.org/ns/ttml"></tt>';
           subtitles = ImscSubtitles(mediaPlayer, true, mockParentElement, mockMediaSources, {});
 
-          expect(imscMock.fromXML).toHaveBeenCalledWith(loadUrlStubResponseText);
+          expect(imscMock.fromXML).toHaveBeenCalledWith('<tt xmlns="http://www.w3.org/ns/ttml"></tt>');
```

Comment:
This test needed to be updated as previously the regex wasn't matching (because of the extra property bit) meaning the split did not remove the XML declaration. Now it does match, it will be removed when doing a `.split`